src/index.js

```diff
@@ -192,21 +192,25 @@ const getHeaders = async (customHeaders = [], current, absolutePath, stats) => {
 		}
 	}
 
-	const defaultHeaders = {
-		'Last-Modified': stats.mtime.toUTCString(),
-		'Content-Length': stats.size,
-		// Default to "inline", which always tries to render in the browser,
-		// if that's not working, it will save the file. But to be clear: This
-		// only happens if it cannot find a appropiate value.
-		'Content-Disposition': contentDisposition(base, {
-			type: 'inline'
-		})
-	};
+	let defaultHeaders = {};
```

Comment:
you could also destructure to the existing const variable but that's a bit ugly 🤷‍♂️